### PR TITLE
YottaDB content for FOSDEM 2021

### PR DESCRIPTION
**Pull request by jsikri94**
Content files for YottaDB stand at FOSDEM

### CODE REVIEW

File: _index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    / SQL-92 access to the database for analytics and reporting, as well as production
+    grade support for access from the Rust language.
+    <br>
+    We have made numerous smaller enhancements such as simpler installation, enhanced 
+    troubleshooting tools, performance enhancements,
+    and of course (like every software project) bug fixes.
+  </p>
+showcase: |
+  <p>
+    Based on a mature code base (in daily production use since 1986, but continuously
```
Comment (ksbhaskar) [inline _index.md]: 1986, but continuously → 1986; continuously

[I don't like the but in this context, and there is another “and” just a few words later.]

File: _index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    invested in and evolving since), YottaDB is a language-agnostic hierarchical key-value
+    database on which all common NoSQL and SQL user cases map well. Its robustness, scalability
```
Comment (ksbhaskar) [inline _index.md]: database on which → database to which

SQL user cases → SQL user cases

File: acid.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ - Compute and debit any applicable service charges.
+ - Create a record to log the transaction.
+
+Consider that application logic executing concurrently with an update to business rules affecting minimum balances and service charges, and a balance inquiry. Each of the three is implemented as an ACID transaction.
+
+**Atomicity** is the property that the entire transaction happens or none of it happens. For example, in the event of a system malfunction, it should not recover to an intermediate state such as between the withdrawal from savings and the deposit to checking.
+
+**Consistency** is the property that the database should never be observable in a state that is inconsistent (the process of course sees its own data that it is manipulating in a transient state).
+
+**Isolation** is the property that each transaction execute and commit as if it were the only transaction active on the system. For example, if the money transfer transaction commits before the transaction that updates business rules.
```
Comment (ksbhaskar) [inline acid.md]: transaction execute and commit as if → transaction executes and commits as if

For example, if the money transfer transaction commits before the transaction that updates business rules → For example, from the point of view of the database state change, the money-transfer commits either before or after the business rules update, even if the logic for both executes concurrently

File: business_continuity.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Business Continuity"
+draft: false
+---
+
+# Business Continuity
+
+While Durability is based on recovering application state from persistent non-volatile storage, business continuity of an application requires recovery of application state even in the face of loss of that non-volatile storage, e.g., the unplanned loss of a data center or a geography. Furthermore, in the event of unplanned loss of a data center or a geography, a mission critical application should continue to be available from another data center.
```
Comment (ksbhaskar) [inline business_continuity.md]: Furthermore, in the event → In the event

File: business_continuity.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Given the strict serialization requirements of demanding applications such as banking core processing, YottaDB provides mechanisms for an application to restore strict serialization after such an outage.
+
+The operation of the primary instance that originates updates is unaffected by the state of any replication connection or secondary instance receiving updates. The instances share no hardware or software resources, and thus one instance is unaffected by any other instance.
+
+---
+**NOTE**
+
+With n instances, it is possible to provide business continuity in the face of at most n-1 unplanned outages. Since the number of instances is always a finite number, absolute business continuity does not exist, and the number of instances is a business decision rather than a technical decision resulting from a YottaDB limit.
```
Comment (ksbhaskar) [inline business_continuity.md]: instances is a business decision → instances becomes a business decision

File: features.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Security
+
+With a philosophy that complexity is the enemy of security, the simply-explained YottaDB security model is based on the mature and easily understood [user-group-world](https://en.wikipedia.org/wiki/File-system_permissions) permissions for processes to access database files. Security can be further enhanced with layered controls like [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux).
+
+## Multi-Language Database Access
+
+{{< figure src="/stands/yottadb/multi-lang-db-access.svg" width="50%" height="50%">}}
+
+While the historic roots of YottaDB are in the M/MUMPS language, YottaDB extends the tight integration of the language with the database to other languages. As C is the lingua franca of programming languages (in that all programming languages have the ability to call a C API), YottaDB’s C language API extends the database engine to all programming languages. In the future, YottaDB LLC intends to create standard bindings from other languages through the C API.
```
Comment (ksbhaskar) [inline features.md]: M/MUMPS → M

In the future, YottaDB LLC intends to create standard bindings from other languages through the C API → Through the C API, bindings (called “wrappers”) exist to natively access YottaDB concurrently also from Go, Perl, and Rust. Community developed wrappers from C++, node.js, and Python are under development and at various levels of usability and maturity, with all expected soon

File: success_stories.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+draft: false
+---
+
+# Success Stories
+
+## University of Antwerp Library
+
+{{< figure src="/stands/yottadb/UoAntwerp_logo.jpg" alt="UoAntwerp Logo" width="50%" height="50%">}}
+
+YottaDB/GT.M powers The University of Antwerp Library catalog, which includes 3.7 million copies of 2.4 million titles, and delivers fast response times and reliable access to users around the world.
```
Comment (ksbhaskar) [inline success_stories.md]: YottaDB/GT.M → YottaDB

File: use_cases.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+Plus, using a single database engine for the edge and the cloud simplifies application design and operational management, as well as allowing data to move seamlessly from one instance to another.
+
+## Health Records
+
+[M](https://en.wikipedia.org/wiki/MUMPS) is a de facto standard for implementing electronic health record applications, and the largest electronic health record systems in the world are written in M, dominating the segment as effectively as SQL dominates accounting and business applications.
+
+Under the GT.M brand, the YottaDB code base has its roots in healthcare as an implementation of M, with its first deployment at the Elvis Presley Trauma Center in Memphis in 1986. YottaDB includes an M language subsystem that is mostly compliant with, and extends the ISO/IEC 11756:1999 and ISO/IEC 15851:1999 standards. The YottaDB/GT.M code base is one of the standard platforms for implementing VistA, ranging from small primary care practices and community clinics such as Oroville Hospital to Hakeem, an electronic health record system for an entire country.
```
Comment (ksbhaskar) [inline use_cases.md]: Under the GT.M brand, the YottaDB → The YottaDB

The YottaDB/GT.M code base → The code base

**Review by ksbhaskar — APPROVED** [review]

File: _index.md (hunk 1 above)
Comment (jsikri94) [reply _index.md]: Should the `and` be removed as well?

Comment (ksbhaskar) [reply _index.md]: I think the remaining and is OK.

**Pull request merged**